crates/dc_bundle/src/legacy_definition/element/vector.rs

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+use serde::{Deserialize, Serialize};
+
+// XXX ColorStop, Transform
```

Comment:
Do you happen to know what this comment is supposed to mean?

Reply:
Not really. @rylin8 Robert do you know what does this comment mean?

Reply:
Not exactly, though usually @iamralpht adds the `// XXX` comments to indicate something we should fix or add support for in the future.